jobs/grafana/templates/create_datasource.erb

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+LOG_DIR=/var/vcap/sys/log/grafana
+MARKERFILE=/var/vcap/sys/run/grafana/datasource_created
+
+<% if_p("grafana.datasource.url") do |datasource_url| %>
+exec &>> ${LOG_DIR}/datasource_create.log
+
+URL="http://<%= p("grafana.admin_username") %>:<%= p("grafana.admin_password") %>@localhost:<%= p("grafana.listen_port") %>/api/datasources"
```

Comment:
I am sorry, I am at home already. But I can take care of the issues tomorrow.

Am 10. November 2015 19:12:59 MEZ, schrieb Alex Suraci <notifications@github.com>:
>> @@ -0,0 +1,40 @@
>> +#!/bin/bash
>> +
>> +LOG_DIR=/var/vcap/sys/log/grafana
>> +MARKERFILE=/var/vcap/sys/run/grafana/datasource_created
>> +
>> +<% if_p("grafana.datasource.url") do |datasource_url| %>
>> +exec &>> ${LOG_DIR}/datasource_create.log
>> +
>> +URL="http://<%= p("grafana.admin_username") %>:<%=
>p("grafana.admin_password") %>@localhost:<%= p("grafana.listen_port")
>%>/api/datasources"
>
>Ran into a few problems with this:
>
>1. My password has funny characters, so they'd need to be urlencoded.
>Maybe use the `-u` flag with `curl` instead; it seems to handle it
>properly.
>2. Would prefer `127.0.0.1` over `localhost`.
>3. My instance is configured with HTTPS, so this would need to go to
>that URI scheme instead (probably with `-k` to trust the local cert).
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/vito/grafana-boshrelease/pull/3/files#r44442343

-- 
Diese Nachricht wurde von meinem Android-Mobiltelefon mit K-9 Mail gesendet.